digiwf-engine/digiwf-engine-service/src/test/java/de/muenchen/oss/digiwf/humantask/process/listener/UserTaskNotificationListenerV02Test.java

```diff
@@ -340,4 +341,175 @@ public void testDelegateTask_WithCandidateUsersAndCandidateGroups() throws Excep
+        final String username = "flash.gordon";
+        DelegateTask task = Mockito.mock(DelegateTask.class);
+        Mockito.when(task.getEventName()).thenReturn("create");
+        Mockito.when(task.getVariable("app_notification_send_assignee")).thenReturn("true");
+        Mockito.when(task.getVariable("app_notification_send_candidate_users")).thenReturn("false");
+        Mockito.when(task.getVariable("app_notification_send_candidate_groups")).thenReturn("false");
+        Mockito.when(task.getVariable("app_task_assignee")).thenReturn(username);
+        when(task.getCandidates()).thenReturn(Collections.<IdentityLink>emptySet());
+
+        DigitalWFProperties properties = Mockito.mock(DigitalWFProperties.class);
+        UserService userService = Mockito.mock(UserService.class);
+        User user = new User();
+        user.setEmail(username + "@muenchen.de");
+        when(userService.getUser(username)).thenReturn(user);
+        MailingService mailingService = Mockito.mock(MailingService.class);
+        RepositoryService repositoryService = Mockito.mock(RepositoryService.class);
+
+        // execute
+        new UserTaskNotificationListener(repositoryService, mailingService, userService, properties).delegateTask(task);
+
+        ArgumentCaptor<MailTemplate> argument = ArgumentCaptor.forClass(MailTemplate.class);
+        verify(mailingService).sendMailTemplateWithLink(argument.capture());
+        assertEquals(user.getEmail(), argument.getValue().getReceivers());
+        assertThat(argument.getValue().getSubject()).isEqualTo("Es liegt eine neue Aufgabe für Sie bereit");
+        assertThat(argument.getValue().getBody()).isEqualTo("Sie haben eine Aufgabe in DigiWF.");
+        assertThat(argument.getValue().getBottomText()).isBlank();
+    }
+
+    /**
+     * Tests if a notification to the assignee is send out with the customized mail subject, body and bottom text.
+     */
+    @Test
+    public void testDelegateTask_WithAssigneeAndCustomizedSubjectBodyAndBottomText() throws Exception {
+        final String username = "flash.gordon";
+        DelegateTask task = Mockito.mock(DelegateTask.class);
+        Mockito.when(task.getEventName()).thenReturn("create");
+        Mockito.when(task.getVariable("app_notification_send_assignee")).thenReturn("true");
+        Mockito.when(task.getVariable("app_notification_send_candidate_users")).thenReturn("false");
+        Mockito.when(task.getVariable("app_notification_send_candidate_groups")).thenReturn("false");
+        Mockito.when(task.getVariable("app_task_assignee")).thenReturn(username);
+        Mockito.when(task.getVariable("MAIL_SUBJECT")).thenReturn("Neue Testaufgabe");
+        Mockito.when(task.getVariable("MAIL_BODY")).thenReturn("Hier kommen Sie zu der neuen Testaufgabe.");
+        Mockito.when(task.getVariable("MAIL_BOTTOM_TEXT")).thenReturn("Viele Grüße");
+        when(task.getCandidates()).thenReturn(Collections.<IdentityLink>emptySet());
```

Comment:
Wenn du einmal den statischen Import nutzt, kann oben komplett Mockito weg. :-)

Reply:
Wie besprochen wird das erstmal so beibehalten, da es in allen Tests angepasst werden muss.